manual/query_builder/idempotence/README.md

```diff
@@ -155,6 +157,19 @@ Statement statement =
 assert statement.isIdempotent();
 ```
 
+However, in rare cases the resulting statement won't be marked idempotent, e.g. if you use a 
+function to select a collection element:
+
+```java
+SimpleStatement statement =
+    update("foo")
+        .removeListElement("l", function("myfunc"))
+        .whereColumn("k").isEqualTo(bindMarker())
+        .build();
+// UPDATE foo SET l=l-[myfunc()] WHERE k=?
+assert !statement.isIdempotent();
+```
+
```

Comment:
👍 I had to refer to the javadocs to remember when terms weren't idempotent, so good thing to mention it in the manual too.